obspy/io/dmx/tests/test_core.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+The obspy_dmx.core test suite.
+"""
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+from future.builtins import *  # NOQA
+
+import os
+import unittest
+
+from obspy import read
+from obspy.core.utcdatetime import UTCDateTime
+from obspy.io.dmx.core import _read_dmx
+
+
+class CoreTestCase(unittest.TestCase):
+    """
+    Test cases for win core interface
+    """
+    def setUp(self):
+        # directory where the test files are located
+        self.path = os.path.join(os.path.dirname(__file__), 'data')
+
+    def test_read_via_obspy(self):
+        """
+        Read files via obspy.core.stream.read function.
+        """
+        filename = os.path.join(self.path, '181223 120000.DMX')
+        # 1
+        st = read(filename)
+        st.verify()
+        st.sort(keys=['channel'])
```

Comment:
the reading operation should be deterministic and no sort should be done here testing results

Reply:
oke